index.js

```diff
@@ -105,7 +155,18 @@ function findById() {}
  *  filterByGenre(movies, "Horror")
  *  //> []
  */
-function filterByGenre() {}
+function filterByGenre(movies, genre) {
+  let getOut = []
```

Comment:
Funny, but I still think we could've called this something like `moviesByGenre`. Happy to see you're having fun with it though.